nifi-nar-bundles/nifi-groovyx-bundle/pom.xml

```diff
@@ -39,8 +39,14 @@
             </dependency>
             <dependency>
                 <groupId>org.codehaus.groovy</groupId>
-                <artifactId>groovy-all</artifactId>
-                <version>2.4.11</version>
+                <artifactId>groovy-json</artifactId>
```

Comment:
See previous comment.

Reply:
Changes to `nifi-groovyx-bundle/pom.xml` will be kept in sync with the decision regarding [this comment](https://github.com/apache/nifi/pull/3547#discussion_r297397319).